cc/algorithms/numerical-mechanisms.h

```diff
@@ -75,7 +75,8 @@ class NumericalMechanism {
   virtual int64_t MemoryUsed() = 0;
 
   virtual base::StatusOr<ConfidenceInterval> NoiseConfidenceInterval(
-      double confidence_level, double privacy_budget) {
+      double confidence_level, double privacy_budget,
+      double noised_result = 0) {
```

Comment:
Per the Google c++ style guide, please use overrides instead of default arguments for virtual functions (https://google.github.io/styleguide/cppguide.html#Default_Arguments)

Reply:
Thanks for catching that! I edited the code to follow Google's C++ style guide. :)